lib/Dredd.js

```diff
@@ -250,5 +249,5 @@ Is the provided path correct?
   }
 }
 
+
 module.exports = Dredd;
-module.exports.options = options;
```

Comment:
Is this part of the public API / a breaking change?

Reply:
Hmm, you're right. While it's undocumented, this is breaking, I'll amend the message. I want it removed as I don't see any value in having it exported to users like this, the options definitions are internal to Dredd.